sdks/python/src/opik/id_helpers.py

```diff
@@ -7,37 +7,30 @@
 
 
 def uuid4_to_uuid7(user_datetime: datetime, user_uuid: str) -> uuid.UUID:
-    """Convert an UUID4 in a UUID7 in a naive way"""
-    last = [0, 0, 0, 0]
-
-    # Use a hardcoded timestamp in ns for now
-    ns = UUIDV7_NS
```

Comment:
I assume we no longer need it. @Lothiraldan, do you use this in the dify integration?

Reply:
No we don't use it https://github.com/langgenius/dify/blob/main/api/core/ops/opik_trace/opik_trace.py

Reply:
@jverre lets remove this constant then.